var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -93,7 +93,10 @@ class Gcc(AutotoolsPackage):
     depends_on('mpfr@2.4.2:')
```

Comment:
I am tempted to write the currently working latest majors here, until someone tests and actively bumps upcoming/newer dependency releases with older (fixed!) GCC tags to:
```
depends_on('gmp@4.3.2:6')
depends_on('mpfr@2.4.2:4')
depends_on('mpc@0.8.1:1', when='@4.5:')
```

Reply:
Yes, should limit: https://github.com/spack/spack/issues/6902#issuecomment-433030376